python/SWESimulators/CTCS.py

```diff
@@ -40,11 +39,11 @@ class CTCS(Simulator.Simulator):
     """
 
     def __init__(self, \
-                 cl_ctx, \
+                 gpu_ctx, \
                  H, eta0, hu0, hv0, \
                  nx, ny, \
                  dx, dy, dt, \
-                 g, f, r, A=0.0, \
+                 g, f, r, A, \
```

Comment:
Is the default removed by design or by accident?

Reply:
Accident. Will fix.